cpu/nrf52/include/periph_cpu.h

```diff
@@ -43,9 +43,6 @@ extern "C" {
 #define SPI_MOSISEL         (dev(bus)->PSEL.MOSI)
 #define SPI_MISOSEL         (dev(bus)->PSEL.MISO)
 #ifndef CPU_MODEL_NRF52840XXAA
-#define UART_PIN_RTS        (GPIO_UNDEF)
```

Comment:
IIRC, this was required for nrf51/nrf52832 boards to build nicely if the defines don't exits.

Reply:
but its not needed anymore with the restructured code...

Reply:
`UART_HWFLOWCTRL` is still required I think.